fendermint/vm/interpreter/src/fvm/exec.rs

```diff
@@ -95,7 +95,12 @@ where
         let method_num = msg.method_num;
         let gas_limit = msg.gas_limit;
 
-        let (apply_ret, emitters) = state.execute_explicit(msg)?;
+        // system actor executes implicitly
```

Comment:
```suggestion
        // Explicit execution requires `from` to be an account kind, which the System Actor is not, 
        // so it can only be executed implicitly.
```